src/main/java/codeu/model/data/User.java

```diff
@@ -14,61 +14,109 @@
 
 package codeu.model.data;
 
+import codeu.model.store.basic.UserStore;
+
 import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
 import java.util.UUID;
 
 /**
  * Class representing a registered user.
  */
 public class User {
-    private final UUID id;
-    private final String name;
-    private final String passwordHash;
-    private final Instant creation;
-
-    /**
-     * Constructs a new User.
-     *
-     * @param id           the ID of this User
-     * @param name         the username of this User
-     * @param passwordHash the password hash of this User
-     * @param creation     the creation time of this User
-     */
-    public User(UUID id, String name, String passwordHash, Instant creation) {
-        this.id = id;
-        this.name = name;
-        this.passwordHash = passwordHash;
-        this.creation = creation;
-    }
-
-    /**
-     * Returns the ID of this User.
-     */
-    public UUID getId() {
-        return id;
-    }
-
-    /**
-     * Returns the username of this User.
-     */
-    public String getName() {
-        return name;
-    }
-
-    /**
-     * Returns the password hash of this User.
-     */
-    public String getPasswordHash() {
-        return passwordHash;
-    }
-
-    /**
-     * Returns the creation time of this User.
-     */
-    public Instant getCreationTime() {
-        return creation;
-    }
 
+  private final UUID id;
+  private final String name;
+  private final String passwordHash;
+  private final Instant creation;
+  private Set<String> friends;
+  private UserStore userStore;
+
+  /**
+   * Constructs a new User.
+   *
+   * @param id the ID of this User
+   * @param name the username of this User
+   * @param passwordHash the password hash of this User
+   * @param creation the creation time of this User
+   * @param friends a set with the usernames of the users friends
+   */
+  public User(UUID id, String name, String passwordHash, Instant creation, Set<String> friends) {
+    this.id = id;
+    this.name = name;
+    this.passwordHash = passwordHash;
+    this.creation = creation;
+    this.friends = friends;
+    userStore = UserStore.getInstance();
+  }
```

Comment:
There's a problem here because Gavin and I  just merged our changes to the User object adding language and bio as a parameter. I'm not sure why there aren't conflicts anymore, but this would erase all of the changes we added.